### PR TITLE
chore: make internal methods snake case

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ from playwright import sync_playwright
 with sync_playwright() as p:
     for browser_type in [p.chromium, p.firefox, p.webkit]:
         browser = browser_type.launch()
-        page = browser.newPage()
+        page = browser.new_page()
```
Comment (kumaraditya303) [inline README.md:55]: PR #378

File: tests/async/test_dispatch_event.py [hunk 2]
```diff
@@ -125,7 +125,7 @@ async def test_should_be_atomic(selectors, page, utils):
                 return result;
             },
             queryAll(root, selector) {
-                const result = Array.from(root.querySelectorAll(selector));
```
Comment (mxschmitt) [inline tests/async/test_dispatch_event.py:128]: should not be changed afaik.

File: tests/assets/react/react-dom@16.13.1.production.min.js [hunk 3]
```diff
@@ -232,7 +232,7 @@ b,b.pendingProps.children,c),b.child;case 12:return T(a,b,b.pendingProps.childre
...
-b);df(a,b)}};sd=function(a,b,c){switch(b){case "input":Dd(a,c);b=c.name;if("radio"===c.type&&null!=b){for(c=a;c.parentNode;)c=c.parentNode;c=c.querySelectorAll("input[name="+JSON.stringify(""+b)+'][type="radio"]');for(b=0;b<c.length;b++){var d=c[b];if(d!==a&&d.form===a.form){var e=ae(d);if(!e)throw Error(k(90));Gf(d);Dd(d,e)}}}break;case "textarea":Lf(a,c);break;case "select":b=c.value,null!=b&&hb(a,!!c.multiple,b,!1)}};(function(a,b,c,d){ee=a;eg=b;vd=c;vf=d})(Qh,function(a,b,c,d,e){var f=p;p|=4;
+b);df(a,b)}};sd=function(a,b,c){switch(b){case "input":Dd(a,c);b=c.name;if("radio"===c.type&&null!=b){for(c=a;c.parentNode;)c=c.parentNode;c=c.query_selector_all("input[name="+JSON.stringify(""+b)+'][type="radio"]');for(b=0;b<c.length;b++){var d=c[b];if(d!==a&&d.form===a.form){var e=ae(d);if(!e)throw Error(k(90));Gf(d);Dd(d,e)}}}break;case "textarea":Lf(a,c);break;case "select":b=c.value,null!=b&&hb(a,!!c.multiple,b,!1)}};(function(a,b,c,d){ee=a;eg=b;vd=c;vf=d})(Qh,function(a,b,c,d,e){var f=p;p|=4;
```
Comment (mxschmitt) [inline tests/assets/react/react-dom@16.13.1.production.min.js:235]: revert this line.

File: tests/assets/sectionselectorengine.js [hunk 4]
```diff
@@ -5,6 +5,6 @@
     return root.querySelector('section');
   },
   queryAll(root, selector) {
-    return Array.from(root.querySelectorAll('section'));
+    return Array.from(root.query_selector_all('section'));
```
Comment (mxschmitt) [inline tests/assets/sectionselectorengine.js:8]: I think  that was changed unintentionally.

File: tests/async/test_navigation.py [hunk 5]
```diff
@@ -541,11 +541,11 @@ async def test_wait_for_nav_should_work_with_dom_history_back_forward(page, serv
     await page.goto(server.EMPTY_PAGE)
     await page.set_content(
         """
-      <a id=back onclick='javascript:goBack()'>back</a>
-      <a id=forward onclick='javascript:goForward()'>forward</a>
+      <a id=back onclick='javascript:go_back()'>back</a>
+      <a id=forward onclick='javascript:go_forward()'>forward</a>
       <script>
-        function goBack() { history.back(); }
```
Comment (mxschmitt) [inline tests/async/test_navigation.py:547]: nit: revert too theses function renames to have the same as upstream.

File: tests/async/test_queryselector.py [hunk 6]
```diff
@@ -136,7 +136,7 @@ async def test_selectors_register_should_handle_errors(selectors, page: Page, ut
         return root.querySelector('dummy');
       },
       queryAll(root, selector) {
-        return Array.from(root.querySelectorAll('dummy'));
+        return Array.from(root.query_selector_all('dummy'));
```
Comment (mxschmitt) [inline tests/async/test_queryselector.py:139]: revert?

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**